docs/en/development-guide.md

```diff
@@ -0,0 +1,82 @@
+# Development guide
+
+## Run your first build
```

Comment:
Just checking, do we want to mention a process to fork repo and set up a signature for commit somewhere? 
We might be able to skip that if that's very common for OSS but I don't have much experience so not sure about that.

Reply:
We should cover the process. I added sections.